testutils/channel_opts.go

```diff
@@ -66,6 +66,12 @@ type ChannelOpts struct {
 	// negative values are treated as a single run.
 	RunCount int
 
+	// CheckFrameLeaks indicates whether we should check for frame leaks or not.
+	// This causes the same tests to be run twice, first with the default frame pool,
+	// then with the recording frame pool, which will double the amount of time it takes
+	// for the test.
+	CheckFrameLeaks bool
```

Comment:
nit: this isn't just checking for frame leaks, but whether the frame pool is being used correctly (e.g., detects multiple release), so maybe `CheckFramePooling`?